src/Behat/Testwork/Hook/Call/RuntimeSuiteHook.php

```diff
@@ -37,10 +38,19 @@ public function __construct($scopeName, $filterString, $callable, $description =
         parent::__construct($scopeName, $filterString, $callable, $description);
 
         if ($this->isAnInstanceMethod()) {
+            if (is_array($callable)) {
+                $className = $callable[0];
+                $methodName = $callable[1];
+            } else {
+                $reflection = new ReflectionMethod($callable);
+                $className = $reflection->getDeclaringClass()->getShortName();
+                $methodName = $reflection->getName();
+            }
+
```

Comment:
Now that this code is more complex and duplicated in RuntimeFeatureHook, might it be worth pulling it up to a parent class as something like `$this->throwIfInstanceMethod($callable)`?

Reply:
Good shout, done!